### DISCUSSION TITLE
Lavastoviglie senza led

### DISCUSSION THREAD

**Discussion by nromagna** [indomus/forum#1947] [HUB: Home Assistant]
Mannaggi a me... ho acquiostato una nuova lavastoviglie ma è senza il led quando è in funzione.
Chiedevo esiste un Sonoff con controllo di potenza, ossia se la lavastoviglie è in funzione si accende una lampadina...
Grazie.

**Comment by nicolacampagnolo** [comment]
Bisogna capire se la lavastoviglie durante il ciclo assorbe sempre oltre un tot di watt. Da lì puoi creare una automazione senza problemi che quando in funzione il led è acceso....
